cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/verifier/SlangContentVerifier.java

```diff
@@ -40,6 +34,7 @@
     private final static Logger log = Logger.getLogger(SlangContentVerifier.class);
 
     private String[] SLANG_FILE_EXTENSIONS = {"sl", "sl.yaml", "sl.yml"};
+    private static final String[] PROPERTIES_FILE_EXTENSIONS = {"prop.sl"};
```

Comment:
Why do you need to hold this constant twice (also in TestCasesYamlParser)?
You should hold it once and refer to the same constant